components/sysutils/net-snmp/Makefile

```diff
@@ -91,16 +90,19 @@ PYTHON_ARGS =		--basedir=/usr/bin
 CONFIGURE_OPTIONS  +=	--with-python-modules="$(PYTHON_ARGS)"
 
 # Only build 64 bit Perl support
-PERL_ARGS =	DESTDIR=$(PROTO_DIR) INSTALLDIRS=vendor
-CONFIGURE_OPTIONS.32  +=  PERLPROG="$(PERL.5.22)"
+PERL_ARGS =			DESTDIR=$(PROTO_DIR) INSTALLDIRS=vendor
 CONFIGURE_OPTIONS.32  +=	--disable-embedded-perl
-CONFIGURE_OPTIONS.32  +=  --with-perl-modules="$(PERL_ARGS)"
-CONFIGURE_OPTIONS.64  +=	PERLPROG="$(PERL.5.24)"
+CONFIGURE_OPTIONS.32  +=	--without-perl-modules
```

Comment:
Why do we avoid building perl modules?

Reply:
It did not compile with --with-perl-modules. But I found the failure in a Makefile.PL and created a patch for it. It compiles now with the perl modules.